noxfile.py

```diff
@@ -420,6 +419,10 @@ def test(session: nox.Session, execution_group: ExecutionGroup) -> None:
     install_deps(session, execution_group=execution_group)
 
     pytest_args = ["--cov", "--cov-context=test"]
+    if execution_group.experimental:
+        pytest_args.append(
+            "-Wonce"
+        )  # this will override what we set in pyproject, but not be overridden by the cli
```

Comment:
```suggestion
    if execution_group.experimental:
        # don't turn warnings into errors
        # (this will override what we set in pyproject, but not be overridden by the cli)
        pytest_args.append("-Wdefault")
```